autogen.pl

```diff
@@ -776,6 +776,43 @@ sub find_and_delete {
 # Find a specific executable and ensure that it is a recent enough
 # version.
 
+sub check_m4_gnu_extensions {
+    verbose "Checking whether m4 supports the gnu extensions\n";
+    my $m4 = $ENV{'M4'};
+    $m4 = "m4" unless (defined $m4);
+    my $result = `$m4 -g config/check_m4_gnu_extension.m4`;
+    if ($? != 0) {
+        $result = `$m4 config/check_m4_gnu_extension.m4`;
```

Comment:
I think you need to check the exit status of this command, too (e.g., use `OPAL_LOG_COMMAND` here, too).